storage/jellyfish-merkle/src/tree_cache/mod.rs

```diff
@@ -268,17 +265,17 @@ where
     }
 }
 
-impl<'a, R, V> Into<(Vec<HashValue>, TreeUpdateBatch<V>)> for TreeCache<'a, R, V>
+impl<'a, R, V> From<TreeCache<'a, R, V>> for (Vec<HashValue>, TreeUpdateBatch<V>)
```

Comment:
What lint was this from?

Reply:
This one, new in 1.51: https://rust-lang.github.io/rust-clippy/master/#from_over_into